dropwizard-validation/src/main/java/io/dropwizard/validation/ConstraintViolations.java

```diff
@@ -25,21 +20,7 @@ private ConstraintViolations() { /* singleton */ }
     }
 
     public static <T> String validationMethodFormatted(ConstraintViolation<T> v) {
```

Comment:
What do you think about removing this method completely and using `v.getMessage()` explicitly instead? After this change, there isn't any formatting in the method, so its name now looks a little confusing.